### PR TITLE
[Frontend] feat: add WDetailsImage molecule

### PR DESCRIPTION
**Pull request by joseph-robles**
# DetailsImage Molecule
Welcome to the **DetailsImage molecule** for this project.

## Structure
This is the **DetailsImage molecule** index file in the molecules folder: 

image

### Components
We've adopted the *Atomic design* methodology, utilizing only one atom for this molecule:
1. CircleIcon

image

## Screenshot
Here's a sample screenshot of the **DetailsImage molecule**:

image

## Route 
To visualize the **DetailsImage molecule**, simply navigate to the testing URL [http://localhost:3000/test](http://localhost:3000/test) after launching the server using `npm run dev` .

### CODE REVIEW

**Comment by joseph-robles** [comment]
@eds000n Please note that there are 104+ unexpected extra lines from our test page due to me updating it using a Linter so most apostrophes changed to quotation marks, also some spacing was automatically updated. The ***only real changes*** are the **44** new lines from the index.tsx from my new DetailsImage molecule, plus **1** line: `export { default as WDetailsImage } from "./molecules/DetailsImage/index";` from `components/index.tsx`

**Review by RicseDev — APPROVED** [review]

**Review by Anthonyricse — APPROVED** [review]

File: frontend/src/components/molecules/DetailsImage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+interface WDetailsImageProps {
+  accountName: string;
+  op: string;
+  icon: React.ComponentType;
+}
+
+const WDetailsImage: React.FC<WDetailsImageProps> = ({
+  accountName = "Jane Doe",
+  op = "janedoe123",
```
[the remark this answers is not in the document]
Comment (RicseDev) [reply frontend/src/components/molecules/DetailsImage/index.tsx:12]: use accountName , op should other name  , icon

File: frontend/src/components/molecules/DetailsImage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    lineHeight: "19px",
+    color: "#000",
+  };
+  const pStyleOP = {
+    color: "rgba(0, 0, 0, 0.40)",
+  };
+
+  return (
+    <div style={divStyle}>
+      <WCircleIcon iconSize={30} typeColor={"secondary"} icon={IconComponent} />
```
Comment (RicseDev) [inline frontend/src/components/molecules/DetailsImage/index.tsx:35]: replaceIconComponent by icon

File: frontend/src/components/molecules/DetailsImage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <div>
+        <p style={divStyleText}>{accountName}</p>
+        <p style={{ ...divStyleText, ...pStyleOP }}>Posteado por {op}</p>
+      </div>
+    </div>
+  );
+};
+
+export default WDetailsImage;
```
Comment (RicseDev) [inline frontend/src/components/molecules/DetailsImage/index.tsx:44]: use default props 
```jsx
WDetailsImage.defaultProps = {
  accountName: "Jane Doe",
  op: "janedoe123",
  icon: React.Component,
};

File: frontend/src/app/test/page.tsx [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
+import CheckIcon from "@mui/icons-material/Check";
+import AllInclusive from "@mui/icons-material/AllInclusive";
 import WTag from "@/components/atoms/Tag/tag";
-import SearchIcon from '@mui/icons-material/Search';
-import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
-import MailOutlineIcon from '@mui/icons-material/MailOutline';
-import CottageOutlinedIcon from '@mui/icons-material/CottageOutlined';
+import SearchIcon from "@mui/icons-material/Search";
+import NotificationsNoneIcon from "@mui/icons-material/NotificationsNone";
+import MailOutlineIcon from "@mui/icons-material/MailOutline";
```
Comment (AnthonyRicse98) [inline frontend/src/app/test/page.tsx:12]: linter , fix your autoformatter

File: frontend/src/components/molecules/DetailsImage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  const pStyleName = {
+    color: "rgba(0, 0, 0, 0.40)",
+  };
+
+  return (
+    <div style={divStyle}>
+      <WCircleIcon iconSize={30} typeColor={"secondary"} icon={icon} />
+      <div>
+        <p style={divStyleText}>{accountName}</p>
+        <p style={{ ...divStyleText, ...pStyleName }}>Posteado por {name}</p>
```
Comment (eds000n) [inline frontend/src/components/molecules/DetailsImage/index.tsx:48]: use i18n

**Review by eds000n — APPROVED** [review]

**Pull request merged**